### ISSUE TITLE
The model?

### ISSUE THREAD

**Issue by arianaa30** [KolatimiDave/Open-Vino-Action-Recognition#1]
Hi,
Nice repo. How to access and try the model?

**Comment by KolatimiDave** [comment]
Hi, [here](https://docs.openvinotoolkit.org/2021.1/omz_models_intel_action_recognition_0001_description_action_recognition_0001.html) is the link to the model documentation, the models are available as pretrained model on Intel openvino.

**Comment by arianaa30** [comment]
Thanks. Do you know how to get the model and run them? Any samples?

**Comment by KolatimiDave** [comment]
Yeah the models are available as pretrained models in openvino, so you have to download openvino, on how to run them, you can follow my repo on [Face Detection](https://github.com/KolatimiDave/Face-Detection-with-Intel-OPENVINO).
